main.tex

```diff
@@ -198,7 +198,7 @@ \subsection{Astropy development model}
 with a set of possible labels that help classify and organize the issues.
 The development workflow is detailed in the \astropypkg
 documentation.\footnote{\emph{How to make a code contribution},
-\url{http://docs.astropy.org/en/stable/development/workflow/development_workflow.html}}
+\url{http://docs.astropy.org/en/latest/development/workflow/development_workflow.html}}
```

Comment:
NOTE: Since contributors submit changes against `dev`, it makes more sense to point to the `dev` version of the doc.

Reply:
I had a point earlier to point to the stable docs, but agree with @pllim here that the dev docs should point to the latest versions not stable.